documentation/modules/rn-2.2.adoc

```diff
@@ -98,3 +98,7 @@ If a Plan CR references storage, network, or VMs by name instead of by ID, the r
 .Log archive file includes logs of a deleted migration plan or VM
 
 If you delete a migration plan and then run a new migration plan with the same name or if you delete a migrated VM and then remigrate the source VM, the log archive file created by the {project-short} web console might include the logs of the deleted migration plan or VM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764[*BZ#2023764*])
+
+.If a target VM is deleted during migration, its migration status is "Succeeded" in the "Plan" CR
```

Comment:
Looking at the examples above, Plan does not have to be within quotes

Reply:
@mguetta1 
This arises from my misunderstanding of requirements. We are not allowed to use backticks in headers. However, a "dot header" (.Header) is not considered a header in the same sense, so there should be markup here.

@sbeskin-redhat 
Change the quotes to backticks in this line.